puppeteer-jest/jest-puppeteer.config.js

```diff
@@ -0,0 +1,19 @@
+const caps_chrome = {
```

Comment:
Can we add multiple configurations here?

Reply:
we wont be able to invoke them @ankur-lt